kubemarine/procedures/check_iaas.py

```diff
@@ -288,6 +288,30 @@ def system_distributive(cluster):
         
         tc.success(results=", ".join(detected_supported_os))
 
+def check_kernel_version(cluster):
+    """
+    This method compares the linux kernel version with the bad version
+    """
+    with TestCase(cluster.context['testsuite'], '015', "Software", "Kernel version") as tc:
+        bad_results = []
+        bad_kernel_ubuntu = ['5.4.0-132-generic']
+        bad_kernel_centos = []
+        group = cluster.nodes['all']
+        result_group = group.sudo('uname -r && cat /etc/os-release', warn=True)
```

Comment:
warn=True will not throw an exception if the command exits with non-zero code. On the one hand you are trying to handle such case, but below you subscribe result[1] which is unsafe if the command exits abnormally and not produces stdout. So I think we need to either raise error, or properly handle abnormal exiting.

Reply:
not actual